firefox-ios/Client/Frontend/UserContent/UserScripts/AllFrames/AtDocumentEnd/ContextMenu.js

```diff
@@ -20,8 +20,13 @@ window.__firefox__.includeOnce("ContextMenu", function() {
 
     var data = {};
 
-    data.touchX = evt.changedTouches[0].pageX - window.scrollX;
-    data.touchY = evt.changedTouches[0].pageY - window.scrollY;
+    if (evt.changedTouches) {
+      data.touchX = evt.changedTouches[0].pageX - window.scrollX;
+      data.touchY = evt.changedTouches[0].pageY - window.scrollY;
+    } else {
+      data.touchX = evt.pageX - window.scrollX;
+      data.touchY = evt.pageY - window.scrollY;
+    }
```

Comment:
Minor nit, we could simplify this a bit:
```suggestion
    const triggeringEvent = evt.changedTouches?.[0] ?? evt;
    data.touchX = triggeringEvent.pageX - window.scrollX;
    data.touchY = triggeringEvent.pageY - window.scrollY;
```